.github/workflows/deploy-bemo.yml

```diff
@@ -0,0 +1,56 @@
+name: Deploy bemo
+
+# TODO: add some sort of production trigger
```

Comment:
hm just occurred to me that the bemo server deploys should actually be synchronized with the sdk releases. So rather than calling the env that deploys from main 'staging' we should maybe call it 'canary'?

Reply:
ohh that's a good point!